### PR TITLE
Update Roadmap Doc

### PR DESCRIPTION
**Pull request by MuyangAmigo**

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -1,41 +1,42 @@
...
 
-**Ship with confidence:** Teams Toolkit is enterprise-ready to help developers and IT administrators deliver, manage, and monitor apps with confidence.
+- **Ship with confidence:** Teams Toolkit is enterprise-ready to help developers and IT administrators deliver, manage, and monitor apps with confidence.
 
-#### Q3-Q4 2022
+## Q3-Q4, 2023
 
-##### Teams Toolkit General Availability
-We’re planning to make Teams Toolkit for Visual Studio generally available later this year in Visual Studio 2022.
+- **Extend Microsoft 365 Copilot**: Teams Toolkit for Visual Studio and Visual Studio Code will soon let users quickly build a plugin to extend Microsoft 365 Copilot. This [demonstration](https://www.youtube.com/watch?v=6ZNi1GDxvf0&ab_channel=Microsoft365Developer) shows how simple it will be to build a plugin to extend Microsoft 365 Copilot using the Teams Toolkit extension in Visual Studio Code. This is a preview of the experience you will have when we release our first public preview coming soon.
```
Comment (sffamily) [inline ROADMAP.md:19]: put brackets around "coming soon" ?

File: ROADMAP.md [hunk 2]
```diff
@@ -1,41 +1,42 @@
...
+- **Continuous improvements of templates and samples**: App templates and samples continue to be the key starting point for developers to learn how to build apps for Teams and Microsoft 365. We will continue to simplify the app templates and improve the usability of sample app gallery as we incrementally adding more samples to it.
```
Comment (sffamily) [inline ROADMAP.md:20]: Is "Teams and Microsoft 365" supposed to be "Microsoft 365 including Teams"?

File: ROADMAP.md (hunk 1 above)
Comment (MuyangAmigo) [reply ROADMAP.md:19]: Updated.

File: ROADMAP.md (hunk 2 above)
Comment (MuyangAmigo) [reply ROADMAP.md:20]: Updated.

**Review by sffamily — APPROVED** [review]

**Pull request merged**